### PR TITLE
Update Maintain Errors

### LINKED ISSUE #880: Add "Transaction is temporarily banned" to errors
https://wiki.polkadot.network/docs/en/maintain-errors

tx is already in pool, try on different node or just wait

### PR DESCRIPTION
**Pull request by krichard410**
Closes #880

### CODE REVIEW

File: docs/maintain-errors.md [hunk 1]
```diff
@@ -38,11 +38,12 @@ table below.
...
+| Payment            | Unable to pay for TX fee.                                                                                    | You might not have enough free balance to cover the fee this transaction would incur.                                                                                                                                                                                                                                                                                                     |
+| Temporarily banned | The transaction is temporarily banned.                                                                       | The tx is already in pool. Try on a different node and wait.                                                                                                                                                                                                                                                                                                                              |
```
Comment (laboon) [inline docs/maintain-errors.md]:
```suggestion
| Temporarily banned | The transaction is temporarily banned.                                                                       | The tx is already in pool. Either try on a different node, or wait to see if the initial transaction goes through.                                                                                                                                                                                                                                                                                                                              |
```

**Review by laboon — CHANGES_REQUESTED** [review]

File: docs/maintain-errors.md (hunk 1 above)
Comment (krichard410) [reply docs/maintain-errors.md]: What change is new in this suggestion? I don't see a difference 🤔

Comment (laboon) [reply docs/maintain-errors.md]: Scroll to the right =)

Comment (krichard410) [reply docs/maintain-errors.md]: AH i didn't see the scrollbar 😂

**Pull request merged**